doc/margin-calibration-howto.md

```diff
@@ -0,0 +1,44 @@
+# How to calibrate the label margins of DYMO printers
+
+Each model of DYMO printer has a stationary print head of a given width.
+Most DYMO printers support 6mm, 9mm, and 12mm D1 tapes
+and have a 64 pixel print head with a 9mm printable area.
+
+The LabelManager PC II also supports 19mm and 24mm D1 tapes
+and has a 128 pixel print head with a 19mm printable area.
+
```

Comment:
I don't know if you want to list them all, but this page lists the tape options of several printers:
https://www.labelcity.com/dymo-d1-label-tape-compatibility-guide

Just mentioning that the PC II is not the only one to support 24mm